app/src/main/res/layout/emoji_tile.xml

```diff
@@ -19,5 +19,5 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center_vertical"
         android:layout_marginLeft="8dp"
-        android:text="TextView" />
+        android:text="@string/text_view" />
```

Comment:
I don't think putting this into `string.xml` is necessary as we only put strings there for two reasons:
1)To make switching between languages easier
2)Multiple use cases of the same string
@kunall17  what are your thoughts on this?